ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -1809,6 +525,7 @@
 /* Begin XCBuildConfiguration section */
 		00E356F61AD99517003FC87E /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 047663CF32A867699B13342E /* Pods-ZulipMobileTests.debug.xcconfig */;
```

Comment:
What led to this line (and the similar ones below) being here?

It looks like a useful mechanism. I'm curious about it partly because I can't find anything in the Xcode UI that corresponds to it. There's another reference to the same file in the "Pods" group, and that appears in the left sidebar... but when I go to the ZulipMobile target, then to the Build Settings tab which seems to be the thing this affects, I can't find anywhere that says this file is consulted. Though I *do* see there the setting values that reflect this file being taken into account.

Was this in some install instructions? Or added by something automatic? Or something else? :slightly_smiling_face:

Reply:
Answered this at https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/cocoapods/near/846635.